custom/ewsghana/api.py

```diff
@@ -134,16 +134,11 @@ class EWSApi(APISynchronization):
     ]
     PRODUCT_CUSTOM_FIELDS = []
 
-    def _create_location_type_if_not_exists(self, supply_point, location):
-        parent = self._make_loc_type(name=location.location_type)
-        self._make_loc_type(name=supply_point.type, parent_type=parent)
-
     def _create_location_from_supply_point(self, supply_point, location):
         try:
             sql_location = SQLLocation.objects.get(domain=self.domain, site_code=supply_point.code)
             return Loc.get(sql_location.location_id)
         except SQLLocation.DoesNotExist:
-            self._create_location_type_if_not_exists(supply_point, location)
```

Comment:
I saw that now if location type doesn't exist is created in sync_location method.